docs/Documentation/Applications/amrwind.md

```diff
@@ -1,55 +1,156 @@
-# AMR-Wind
+# ExaWind
```

Comment:
Since the page title is changed can you update it on the Applications chart and in the table of contents?